src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxrResolver.cpp

```diff
@@ -96,7 +96,7 @@ HostFxrResolver::GetHostFxrParameters(
             {
                 errorContext.subStatusCode = 38;
                 errorContext.errorReason = "Application DLL not found. Confirm the application dll is present. Single-file deployments are not supported in IIS.";
-                errorContext.generalErrorType = "ANCM Application DLL Not Found";
+                errorContext.generalErrorType = "ASP.NET Core Module Application DLL Not Found";
```

Comment:
```suggestion
                errorContext.generalErrorType = "Failed to locate ASP.NET Core app";
```

"Application DLL" is a very implementation-y phrase to me. Ok for details, less useful in summaries.